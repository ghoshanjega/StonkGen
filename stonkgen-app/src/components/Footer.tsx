import React from 'react'

export const Footer = () => {
    return (
        <footer className="text-center my-5">
            O_O
        </footer>
    )
}
