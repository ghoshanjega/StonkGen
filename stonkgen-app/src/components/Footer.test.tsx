import React from 'react';
import { shallow } from 'enzyme';
import { Footer } from './Footer';


let wrapped = shallow(<Footer />);

describe('FooterComponent', () => {
  it('should render the Footer Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
});