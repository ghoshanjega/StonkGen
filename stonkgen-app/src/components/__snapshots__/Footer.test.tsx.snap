// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent should render the Footer Component correctly 1`] = `ShallowWrapper {}`;
